PLATFORMS.md

```diff
@@ -62,6 +62,7 @@
 - F5 TMSH
 - F5 Linux
 - Fortinet
+- Pica8
```

Comment:
That's not in alphabetical order like the rest

Reply:
fixed the ordering in 58b45ad